nmap-service-probes

```diff
@@ -16291,6 +16291,27 @@ ports 4533
 # Maybe rigctld also?
 match rotctld m|^get_info: (.*)\nRPRT 0\n| p/Hamlib rotctld/ i/model: $1/
 
+##############################NEXT PROBE##############################
+# Ubiquiti Discovery Protocol
+Probe UDP UbiquitiDiscoveryv1 q|\x01\0\0\0|
+rarity 9
+ports 10001
+
+# Valid response is protocol version (\x01) and cmd (\0) followed
+# by 2 bytes of length then TLV groups
+match ubiquiti-discovery m|^\x01\0.[^\0].{48,}|s p/Ubiquiti Discovery Service/ i/v2 protocol/ cpe:/h:ubnt/ cpe:/a:ubnt/
```

Comment:
Here and in the other one, there's not really a difference between `.{48,}` and `.{48}` other than the first has the potential to do more backtracking (I haven't tested to know if there's really impact, but the simpler syntax seems preferable here.)

Also, the CPE entries are missing product info. I know there's not much more you can add here, but I'm worried about compatibility of other code (our own or others) that consumes the CPE and expects a product name.